src/Sulu/Bundle/MarkupBundle/Tests/Unit/Listener/MarkupListenerTest.php

```diff
@@ -72,7 +73,25 @@ protected function setUp(): void
             $this->response->reveal()
         );
 
-        $this->listener = new MarkupListener(['html' => $this->markupParser->reveal()], ['text/html' => 'html']);
+        $container = new class([
+            'html' => $this->markupParser->reveal(),
+        ]) implements ContainerInterface {
+            public function __construct(private array $services)
+            {
+            }
+
+            public function get($id)
+            {
+                return $this->services[$id];
+            }
+
+            public function has($id): bool
+            {
+                return \array_key_exists($id, this->services);
+            }
+        };
```

Comment:
Maybe this should be generalized or we could pull this from the container if it's a `KernelTestCase`